### PR TITLE
Api connection

### PR DESCRIPTION
**Pull request by cancinos**
Basic java class for performing API's request.

### CODE REVIEW

File: GudBox/app/src/main/java/com/ingsoft/url/gudbox/_API_Connection.java [hunk 1]
```diff
@@ -1,9 +1,52 @@
...
+            connect("api/GetSeeds");
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+    }
+
+    /***
+     * API connection method
+     * @param function_url
+     * @return
```
Comment (durini309) [inline GudBox/app/src/main/java/com/ingsoft/url/gudbox/_API_Connection.java]: Please, write a description of the @param and @return

Comment (cancinos) [inline GudBox/app/src/main/java/com/ingsoft/url/gudbox/_API_Connection.java]: In which format would you like to have this description?

Comment (durini309) [inline GudBox/app/src/main/java/com/ingsoft/url/gudbox/_API_Connection.java]: Mandarin, please.

Joking. For example:
  ```
@param function_url (string) -> used for web request
@return (Boolean) -> true if connected, false if don't
```

Comment (cancinos) [inline GudBox/app/src/main/java/com/ingsoft/url/gudbox/_API_Connection.java]: Done @durini309 .
Review it again and accept/decline the merge, please.

**Pull request merged**